### ISSUE TITLE
Tilgængelighedsproblem med disabled link state

### ISSUE THREAD

**Issue by stil-ux-team** [FSGpilot/frontend-styleguide-components#3]
"disabled link state" er ikke tilgængeligt pga for lav kontrast.

**Comment by Kofoed** [comment]
1.4.3 Contrast (Minimum): The visual presentation of text and images of text has a contrast ratio of at least 4.5:1, except for the following: (Level AA)

Large Text: Large-scale text and images of large-scale text have a contrast ratio of at least 3:1;

Incidental: Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.

Logotypes: Text that is part of a logo or brand name has no minimum contrast requirement.

https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html

**Comment by emiljensengit** [comment]
The disabled link now has a new accessible color.
